### DISCUSSION TITLE
How developer with no background in ai development or GPT model can contribute?

### DISCUSSION THREAD

**Discussion by unkwnownGriot** [arc53/DocsGPT#95] [Q&A] [answered]
hey how are you doing? your project looks amazing and i want to contribute but i don't know where to start i've experience in python backend development withh flask but i've never use it with GPT model and i want to try it. if someone could guide me and take me under his shoulder as a mentor it would be great, thank you

**Comment by dartpain** [comment, marked as answer]
Hey, sounds great, join our discord, we have a very friendly community, I'll give you a quick look around our app, and show you some good first issues that could be interesting! [Discrod](https://discord.gg/n5BX8dh8rU) im @dartpain  there
